services/ResourceServiceProvider/src/Provider/ResourceServiceProvider.php

```diff
@@ -118,49 +123,32 @@ function boot(Application $app) {
    * Part of ControllerProviderInterface
    */
   public function connect(Application $app) {
-    $controllers = $app['controllers_factory'];
+    $ResourceControllers = $app['controllers_factory'];
     //
     // Define routing referring to controller services
     //
-    $controllers->get("/resource/{id}/{child}", "islandora.resourcecontroller:get")
+    $ResourceControllers
       ->convert('id', $app['islandora.idToUri'])
       ->assert('id',$app['config']['islandora']['resourceIdRegex'])
       ->before($app['islandora.hostHeaderNormalize']) 
       ->before($app['islandora.htmlHeaderToTurtle'])
-      ->value('id',"")
+      ->value('id',"");
```

Comment:
Yeah, it's a symphony thing and $id is the uuid which can be empty.During registering/mounting they act as a collection, then after that they become sort of anonymous. I thought it would be nice if i could spare some lines/make it more manageable  by putting all the common middleware there.